### ISSUE TITLE
Binance Coin-M Support

### ISSUE THREAD

**Issue by takami-ml** [MtkN1/pybotters#172]
It would be great if datastore officially supports binance coin-m.

Since coin-m and usd-m have almost same specifications, I believe it's not that time-consuming.
I saw the message that spot binance will be supported in the next version.
I would be very happy if coin-m could be supported at that time.

At this moment, I only need position data and I use this initialization code with `store = BinanceDataStore()`
It seems work as expected.

```
    aws = [
        client.post('/dapi/v1/listenKey'),
        client.get('/dapi/v1/positionRisk'),
    ]
    for f in asyncio.as_completed(aws):
        resp = await f
        data = await resp.json()
        if resp.url.path in ('/dapi/v1/listenKey'):
            store.listenkey = data["listenKey"]
            asyncio.create_task(
                store._listenkey(resp.url, resp.__dict__["_raw_session"])
            )
        elif resp.url.path in ("/dapi/v1/positionRisk"):
            store.position._onresponse(data)
```

**Comment by MtkN1** [comment]
Thanks for the request!

It is true that Binance COIN-M is a major derivative, but DataStore is not implemented.
This is not an implementation problem, just a priority issue.
I will include Binance COIN-M in the next release so that I can respond to your request.

**Comment by MtkN1** [comment]
Closed by #189
